README.md

```diff
@@ -71,7 +71,7 @@ module.exports = [
         },
```

Comment:
I wonder if this is still going to work for videos too?

Reply:
What do you think about this @mettlestudiomatt ?

Reply:
I've added the two video urls "videodelivery.net" and "watch.cloudflarestream.com" to the media-src, we will have to test if these urls work and update if they aren't successful.